assets/src/edit-story/components/panels/sizePosition/panel.js

```diff
@@ -92,6 +92,8 @@ function SizePositionPanel({
     ({ type }) => getDefinitionForType(type).canFlip
   );
 
+  const hasText = selectedElements.find(({ type }) => 'text' === type);
```

Comment:
Use [`array.some()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some). `array.find()` returns the actual element, not just whether it exists or not.